openedx/frontend-app-authoring
# chore: bump frontend-build, remove exceptions for prefer-default-export

## Description

This PR bumps `frontend-build` to pull in the eslint config changes from https://github.com/openedx/eslint-config/pull/164 so that default exports are no longer recommended in single-export files. This lets us remove a huge number of exceptions specified in the code, as you can see in the diff.


## Testing instructions

This should only affect linting; no changes to the behavior of the code. Run `npm run lint` or just check the CI output.
